src/Exceptionless.Core/Mail/SmtpMailSender.cs

```diff
@@ -20,16 +21,27 @@ public class SmtpMailSender : IMailSender {
             message.Headers.Add("X-Auto-Response-Suppress", "All");
             message.Headers.Add("Auto-Submitted", "auto-generated");
 
-            var client = new SmtpClient();
-            if (!String.IsNullOrEmpty(Settings.Current.SmtpHost)) {
-                client.Host = Settings.Current.SmtpHost;
-                client.Port = Settings.Current.SmtpPort;
-                client.EnableSsl = Settings.Current.SmtpEnableSsl;
-                client.Credentials = new NetworkCredential(Settings.Current.SmtpUser, Settings.Current.SmtpPassword);
-            }
+            using (var client = new SmtpClient()) {
```

Comment:
Since this class is a singleton (via DI) is it safe to reuse this client and just create it once and connect once?